modules/serverless-knative-kafka-source-metrics.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * serverless/observability/admin-metrics/serverless-admin-metrics.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-knative-kafka-source-metrics_{context}"]
+= Knative Kafka source metrics
+
+You can use the following metrics to debug and visualize the performance of Kafka sources.
+
+[cols=5*,options="header"]
+|===
+|Metric name
+|Description
+|Type
+|Tags
+|Unit
+
+|`event_count_1_total{job="kafka-source-dispatcher-sm-service", namespace="knative-eventing"}`
+|Number of events dispatched by a Kafka source
+|Counter
+a|
+* `consumer_name` (Kafka source name)
+* `namespace_name` (Kafka source namespace)
+* `name` (Kafka source name)
+* `event_type` (event type)
+* `response_code` (HTTP response code returned by the Kafka source sink service)
+* `response_code_class` (HTTP response code class returned by the Kafka source sink service: 2xx, 3xx, 4xx, 5xx)
+|Dimensionless
+
+|`event_dispatch_latencies_ms_bucket{job="kafka-source-dispatcher-sm-service", namespace="knative-eventing"}`
+|The time spent dispatching an event to a sink
+|Histogram
+a|
+* `consumer_name` (Kafka source name)
+* `namespace_name` (Kafka source namespace)
+* `name` (Kafka source name)
+* `event_type` (event type)
+* `response_code` (HTTP response code returned by the Kafka source sink service)
+* `response_code_class` (HTTP response code class returned by the Kafka source sink service: 2xx, 3xx, 4xx, 5xx)
+|Milliseconds
+
+|`event_processing_latencies_ms_bucket{job="kafka-source-dispatcher-sm-service", namespace="knative-eventing"}`
+|The time spent processing an event
+|Histogram
+a|
+* `consumer_name` (Kafka source name)
+* `namespace_name` (Kafka source namespace)
+* `name` (Kafka source name)
+* `event_type` (event type)
+|Milliseconds
+
+|`kafka_broker_controller_consumer_group_expected_replicas`
```

Comment:
The `kafka_broker_controller_consumer_group_expected_replicas`  and `kafka_broker_controller_consumer_group_ready_replicas` 

are not specific to just Kafka Source,  (since 1.34 they are used by any of the scalable ones, so both KafkaSources and Triggers. ) , so they should either be also copied into the Kafka Broker section, or perhaps moved to some new common section? 

CC: @pierDipi

Reply:
I have added `kafka_broker_controller_consumer_group_expected_replicas` and `kafka_broker_controller_consumer_group_ready_replicas` in the **Knative Kafka broker metrics** section